build-tools/android-toolchain/android-toolchain.projitems

```diff
@@ -30,11 +30,11 @@
       <HostOS>Darwin</HostOS>
       <DestDir>build-tools\$(XABuildToolsVersion)</DestDir>
     </AndroidSdkItem>
-    <AndroidSdkItem Include="platform-tools_r23-macosx.zip">
+    <AndroidSdkItem Include="platform-tools_r26.0.0-rc1-macosx.zip">
       <HostOS>Darwin</HostOS>
       <DestDir>platform-tools</DestDir>
     </AndroidSdkItem>
-    <AndroidSdkItem Include="tools_r24.4.1-macosx.zip">
+    <AndroidSdkItem Include="tools_r25.3.1-macosx.zip">
```

Comment:
I am aware of all those download failures. I have no idea what the correct URLs are.